src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleEngine.java

```diff
@@ -268,13 +274,51 @@ protected void setTransactionIsolation() throws SQLException {
     @Override
     protected void createTable(final DbEntity entity) throws DatabaseEngineException {
 
-        List<String> createTable = new ArrayList<>();
+        // Get default create table statement
+        final String defaultCreateTableStatement = getCreateTableStatement(entity);
+
+        // Check for extra properties
+        final String createTableCompressLOBS = getCompressLobsStatements(entity);
+
+        final String createTableStatement = defaultCreateTableStatement + createTableCompressLOBS;
+
+        for (final String createTable : ImmutableList.of(createTableStatement, defaultCreateTableStatement)) {
+            try (
+                    Statement statement = conn.createStatement()
```

Comment:
How about putting this in a single line, with a _final_
`            try (final Statement statement = conn.createStatement()) {`
(the statement is not being closed...)

Reply:
I'm currently using try-with-resources so the statement is being closed automatically. Anyway, I will do a few changes in this pull-request :)

Reply:
uups
Yes, the "statement is not being closed currently" was meant for line 242 below.
My "copy-paste comment" didn't work very well